### DISCUSSION TITLE
Add ALS gameplay tags to owning actor

### DISCUSSION THREAD

**Discussion by Mydrong** [Sixze/ALS-Refactored#548] [Ideas]
Is there any way we could get all the rotation/locomotion etc tags applied as loose tags to the actor? It would be really nice to be able to use the tags as blockers and stuff on gameplay abilities and effects. 

I am doing this by overriding what functions I can to add them to the actor and then editing the plugin code to add what I cant override but it would be nice to have this as base functionality so I don't have to diverge from the main branch so much

**Comment by Magic-Summer** [comment]
You may be able to use [core-redirects](https://github.com/Sixze/ALS-Refactored/discussions/395#discussioncomment-7428429) instead of editing the plugin code.
